children/schema.py

```
@@ -643,10 +647,11 @@ class Query:
     child = relay.Node.Field(ChildNode)
     child_notes = graphene.Field(ChildNotesNode, id=graphene.ID(required=True))
 
-    def resolve_child_notes(self, info, id):
+    @staticmethod
+    def resolve_child_notes(_parent, info, id):
```

Ruff (0.16.1)

[error] Function argument `id` is shadowing a Python builtin

(A002)